src/Solcore/Frontend/Pretty/SolcorePretty.hs

```diff
@@ -316,8 +321,12 @@ instance Pretty Ty where
     = ppr t1 <+> (text "->") <+> ppr t2
   ppr (TyCon n ts)
     | isTuple n = parens $ commaSep (map ppr ts)
+    | isUnit n = text "()"
```

Comment:
How should unit be spelled in source files? "unit" or "()"?

Reply:
Fixed. Notation is `()`.